consensus/consortium/v2/consortium.go

```diff
@@ -491,6 +491,19 @@ func backOffTime(header *types.Header, snapshot *Snapshot) uint64 {
 		return 0
 	}
 
+	var initialDelay time.Duration
```

Comment:
set `initialDelay = time.Second` here, so we can prevent assigning it in 2 `else` clauses